app02.test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const app = require('./app');
+
+test('Query reguest validation', async () => {
+  const response = await app.inject({
+    method: 'GET',
+    url: '/',
+  });
+
+  expect(response.statusCode).toBe(200);
+  expect(response.payload).toBe('Hello world');
+});
```

Comment:
In this scenario, there is no need to split tests into multiple files. You can place as many tests as you need into a single file as long as it makes sense.
Also, file names should be meaningful. Something like `hello.test.js` or `hello-app.test.js` will work for these tests.

Reply:
Do you want to move the tests to a single file?

Reply:
If I should - I do, but for this case we could leave it in two files.

Reply:
There is no need to separate them. They test the same module.
Usually, you have a file per module that is being tested

Reply:
Ok, understood. I’ll take it in next time - file per module.